packages/manager/modules/pci/src/projects/project/public-gateways/add/add.constants.js

```diff
@@ -0,0 +1,33 @@
+export const PUBLIC_GATEWAYS_READ_MORE_GUIDE = {
+  ALL_GUIDE: {
+    DEFAULT:
+      'https://docs.ovh.com/gb/en/public-cloud/creating-private-network-with-gateway/',
+  },
+};
+
+export const REGIONS_AVAILABILITY_URL =
+  'https://www.ovhcloud.com/en-ie/public-cloud/regions-availability/';
+
+export const AVAILABLE_SUBNET = [
+  '192.168.0.1/24',
+  '10.0.0.1/24',
+  '172.16.0.1/24',
+];
+
+export const PRODUCT_NAME = 'GATEWAY';
+export const PUBLIC_GATEWAY_CATALOG_ADDON_PRODUCT_NAME = 'CLOUD';
+export const PUBLIC_GATEWAY_ADDON_PRODUCT_S = 'publiccloud-gateway-s';
+export const PUBLIC_GATEWAY_ADDON_PRODUCT_M = 'publiccloud-gateway-m';
+export const PUBLIC_GATEWAY_ADDON_PRODUCT_L = 'publiccloud-gateway-l';
+export const DEFAULT_IPVERSION = 4;
+
+export default {
+  PUBLIC_GATEWAYS_READ_MORE_GUIDE,
+  REGIONS_AVAILABILITY_URL,
+  PRODUCT_NAME,
+  PUBLIC_GATEWAY_CATALOG_ADDON_PRODUCT_NAME,
+  PUBLIC_GATEWAY_ADDON_PRODUCT_S,
+  PUBLIC_GATEWAY_ADDON_PRODUCT_M,
+  PUBLIC_GATEWAY_ADDON_PRODUCT_L,
```

Comment:
related to this:
https://github.com/ovh/manager/pull/7193/files#r897706266

```suggestion
  PUBLIC_GATEWAY_ADDONS,
```